src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -19,7 +19,8 @@
         mc:Ignorable="d" 
         Height="474" 
         Width="650"
-        ResizeMode="NoResize">
+        ResizeMode="NoResize"
+        Closed="PreferencesView_OnClosed">
```

Comment:
hmm - theres already a handler for the click button, maybe put it there instead of having two close handling events?

Reply:
Oops, thanks.